src/RBDyn/Jacobian.h

```diff
@@ -232,6 +232,16 @@ class RBDYN_DLLAPI Jacobian
 		const Eigen::Ref<const Eigen::MatrixXd>& jac,
 		Eigen::MatrixXd& res) const;
 
+    /**
+        * Project the jacobian in the full robot parameters vector and add to res.
```

Comment:
I would reformulate because the projection does not really occur, something like:
```Add the projection of the jacobian in the full robot parameters to res.```